<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QLineEdit" name="fromlineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tolineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="arrowpix">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="planemess">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inquireButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="plainIdInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入航班号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bookButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>订票</string>
   </property>
  </widget>
  <widget class="QPushButton" name="userButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
